packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -669,6 +690,16 @@ export class KeybindingWidget extends ReactWidget {
         }
     }
 
+    protected handleContextMenu = (item: KeybindingItem, e: React.MouseEvent | React.KeyboardEvent): void => {
+        const target = e.target as HTMLElement;
+        const domRect = target.getBoundingClientRect();
+        this.contextMenuRenderer.render({
+            menuPath: KeymapsMenus.KEYBINDINGS_WIDGET_CONTEXT_MENU,
+            anchor: { x: domRect.left, y: domRect.bottom },
```

Comment:
`e.nativeEvent`?

Reply:
I'm not sure I understand, should I be using `e.nativeEvent` instead?

Reply:
yes, I thought so, we usually we pass native DOM event as an anchor